### PR TITLE
feat(docker): use shorter prefix

### PR DESCRIPTION
**Pull request by serge1peshcoff**
shorter prefix for discounts backend

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
-# OMS Discounts module
-[![Build Status](https://travis-ci.org/AEGEE/oms-discounts.svg)](https://travis-ci.org/AEGEE/oms-discounts)
-![Codecov branch for master](https://img.shields.io/codecov/c/github/AEGEE/oms-discounts.svg)
+# Discounts module
+[![Build Status](https://travis-ci.org/AEGEE/discounts.svg)](https://travis-ci.org/AEGEE/discounts)
```
Comment (WikiRik) [inline README.md:2]: Maybe update this to CircleCI?

Comment (serge1peshcoff) [reply README.md:2]: Makes sense, will do.

Comment (serge1peshcoff) [reply README.md]: Done.

**Comment by serge1peshcoff** [comment]
Will merge and deploy tomorrow.

**Pull request merged**